lib/matplotlib/font_manager.py

```diff
@@ -261,6 +262,14 @@ def _get_fontconfig_fonts():
     return [Path(os.fsdecode(fname)) for fname in out.split(b'\n')]
 
 
+@lru_cache
+def _get_macos_fonts():
+    """Cache and list the font paths known to ``system_profiler SPFontsDataType``."""
+    d, = plistlib.loads(
+        subprocess.check_output(["system_profiler", "-xml", "SPFontsDataType"]))
```

Comment:
Is this output portable across different system languages?

Reply:
From a quick check, the answer appears to be yes.